pilot/pkg/networking/core/v1alpha3/listener.go

```diff
@@ -1024,19 +1024,11 @@ func buildListener(opts buildListenerOpts) *xdsapi.Listener {
 		})
 	}
 
-	var deprecatedV1 *xdsapi.Listener_DeprecatedV1
-	if !opts.bindToPort {
-		deprecatedV1 = &xdsapi.Listener_DeprecatedV1{
-			BindToPort: proto.BoolFalse,
-		}
-	}
-
```

Comment:
? why are you removing this?

Reply:
the DeprecatedV1 field was deleted in envoy:
vendor/github.com/envoyproxy/go-control-plane/envoy/api/v2/listener/listener.pb.go